doc/bn.tex

```diff
@@ -820,28 +820,32 @@ \subsection{Single Digit}
 \}
 \end{alltt} \end{small}
 
-\subsection{Long Constants}
+\subsection{Int32 and Int64 Constants}
 
-To set a constant that is the size of an ISO C ``unsigned long'' and larger than a single digit the following function
-can be used.
+To set a constant with 32 or 64 bits the following function can be used.
```

Comment:
"the following functions" (plural)

I know I am guilty of it myself, but these type of constructs are a bit awkward. Keep it simple, e.g.:
"The following functions set a constant with a size of 32 or 64 bits."
Or a bit more open ended with:
"The functions to set a constant with a size of 32 or 64 bits are:"

Reply:
fixed